package.json

```diff
@@ -21,7 +21,7 @@
   ],
   "license": "OGL-UK-3.0",
   "dependencies": {
-    "@azure/identity": "^1.5.2",
+    "@azure/identity": "^2.0.0",
```

Comment:
Latest version is `2.0.5`.  We're using this elsewhere so should be fine.

https://www.npmjs.com/package/@azure/identity